canonical-web-and-design/greenhouse-exporter
# Add GitHub checks & Fix all the linting issues

# Summary

Fix linting issues
Add all the commands in package.json
Add GitHub checks

# QA

- `dotrun exec yarn run lint-js`
- `dotrun exec yarn run lint-scss`
- `dotrun exec yarn run lint-python`
- No errors :tada: 
- Wait for GitHub actions and make sure they don't fail
